docs/fundamentals-and-concepts/stellar-data-structures/contracts.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Smart Contracts
+sidebar_position: 50
+---
+
+A smart contract is a programmed set of executable code and state that can be invoked or used on the Stellar network.
```

Comment:
I'll definitely give this a deeper review later but just a thought: it might be worth it to mention that Stellar's smart contract platform is called Soroban. I think we're moving away from saying "Soroban" and saying "smart contracts" more but it might be nice to just mention the name?